### ISSUE TITLE
EditorConfig UI not showing any information

### ISSUE THREAD

**Issue by RichardMarsh** [dotnet/roslyn#54568]
When opening the .editorconfig file from VS2019 16.10.13, the new UI for the editor opens, but nothing is shown

![image](https://user-images.githubusercontent.com/54434905/124298881-14a91980-db5d-11eb-82aa-ace033102fb8.png)

I've created the .editorconfig file in several ways but none of them are yielding any results

Example of .editorconfig that shows nothing in the UI but cleary has values when opening via text editor

![image](https://user-images.githubusercontent.com/54434905/124299046-44582180-db5d-11eb-9ad2-a04069b5c2bb.png)

**Comment by Youssef1313** [comment]
Likely a duplicate of https://github.com/dotnet/roslyn/issues/53081.

cc @jmarolf

**Comment by RichardMarsh** [comment]
> Likely a duplicate of #53081.
> 
> cc @jmarolf

Apologies, I didn’t mention that this is in the root of the solution folder

**Comment by Youssef1313** [comment]
@RichardMarsh It'll probably help if you can upload a zipped repro project.

**Comment by RichardMarsh** [comment]
> @RichardMarsh It'll probably help if you can upload a zipped repro project.

@Youssef1313 apologies for the delay in response. 

Unfortunately I cannot upload the project for work reasons. That said I tried to replicate the problem in another solution that I created from scratch and there was no issue. 

After which I kept working in the original solution and was editing the editorconfig manually. Per chance I checked to see if the UI was working and it did. So the problem resolved itself, but I'm unsure how. 

Note that the solution where the original problem existed is relatively large solution (30+ projects) so that might have been a contributing factor. 

Just thought I'd follow up and let you know.
